frameworks/C++/ULib/benchmark_config.json

```diff
@@ -20,6 +19,24 @@
       "notes": "",
       "versus": ""
     },
+    "json": {
```

Comment:
Hi,

in this way I am able to differentiate the execution context (plaintext has a
greater need for resources than json) so I can tuning better...

Greetings

2015-05-28 19:48 GMT+02:00 Mike Smith <notifications@github.com>:

> In frameworks/C++/ULib/benchmark_config.json
> <https://github.com/TechEmpower/FrameworkBenchmarks/pull/1630#discussion_r31259137>
> :
>
> > @@ -20,6 +19,24 @@
> >        "notes": "",
> >        "versus": ""
> >      },
> > +    "json": {
>
> Why add an entire new test for json when it was already included in the
> default?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TechEmpower/FrameworkBenchmarks/pull/1630/files#r31259137>
> .
>